svg/elements/feDiffuseLighting.json

```
@@ -98,6 +100,8 @@
         },
         "in": {
           "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Attribute/in",
+            "spec_url": "https://drafts.fxtf.org/filter-effects/#feDelement-attrdef-filter-primitive-iniffuseLightingElement",
```

The link doesn't work for me. Is it rather this one?
```suggestion
            "spec_url": "https://drafts.fxtf.org/filter-effects/#element-attrdef-filter-primitive-in",
```

---

Yes, I think I made a bad cut & paste,